core/controllers/editor.py

```diff
@@ -772,3 +774,58 @@ def get(self, exploration_id):
         self.render_json({
             'unresolved_answers': unresolved_answers_with_frequency
         })
+
+
+class LearnerAnswerInfoHandler(EditorHandler):
+    """Handles the learner answer info for an exploration state."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_answer_details
+    def get(self, entity_id, entity_type):
```

Comment:
entity_type should typically come before entity_id

Reply:
Actually, there is a problem with it if I place it earlier then I have to add an else case in the can_edit_exploration acl layer, because can_access_answer_details ACL layer calls can_edit_exploration and can_edit_exploration returns the args in the form of (self, exploration_id, *args), therefore I need to like this.

Reply:
I don't understand the issue, sorry. I'm looking at can_access_answer_details -- why can't you have test_can_access(self, entity_type, entity_id, **kwargs) and have that call can_edit_exploration(self, entity_id, **kwargs) if the entity_type is ENTITY_TYPE_EXPLORATION? You should not need to pass the entity type to can_edit_exploration in that case.

Reply:
Done